packages/cli/src/commands/routes.ts

```diff
@@ -0,0 +1,65 @@
+import {Command, flags} from "@oclif/command"
+import {routes as ServerRoutes} from "@blitzjs/server"
+import {log, table as Table} from "@blitzjs/display"
+
+export class Routes extends Command {
+  static description = "Output Blitz Routes"
+  static aliases = ["r"]
+
+  static flags = {
+    help: flags.help({char: "h"}),
+  }
+
+  getColor(type: string) {
+    switch (type) {
+      case "rpc":
+        return "cyan"
+      case "api":
+        return "magenta"
+      default:
+        return "white"
+    }
+  }
+
+  async run() {
+    const config = {
```

Comment:
Nit: should we add a type annotation for `config` and other places that may be missing it. Not sure how consistent we want to be with it.

Reply:
maybe try to get the type from `@blitzjs/server`?

Several cli commands seem only need `rootFolder` as their param because of config normalization has been done in `@blitzjs/server`

Reply:
If the type is available in `@blitzjs/server` then we should use that

Reply:
got it will import `ServerConfig` from `@blitzjs/server`